src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngineOptions.cs

```diff
@@ -36,6 +36,38 @@ public class RazorViewEngineOptions
         public IList<IFileProvider> FileProviders { get; } = new List<IFileProvider>();
 
         /// <summary>
+        /// Gets the locations where <see cref="RazorViewEngine"/> will search for views.
+        /// </summary>
+        /// <remarks>
+        /// The locations of the views returned from controllers that do not belong to an area.
+        /// Locations are composite format strings (see http://msdn.microsoft.com/en-us/library/txafckwd.aspx),
+        /// which contains following indexes:
```

Comment:
And the page you linked to describes these as index components of format items. Will be more understandable to use "format items" here.